packages/core/src/__tests__/ceramic-query-response.test.ts

```diff
@@ -75,14 +75,14 @@ describe('Response to pubsub queries handling', () => {
 
     // Make sure we have some random valid CIDs to respond to pubsub queries with
     for (let i = 0; i < 10; i++) {
-      const commit = await TileDocument.makeGenesis(ceramic, { foo: i }, null)
+      const commit = await TileDocument.makeGenesis(ceramic.signer, { foo: i }, null)
       const cid = await ceramic.dispatcher.storeCommit(commit)
       cids.push(cid)
     }
   })
 
   beforeEach(async () => {
-    const genesisCommit = await TileDocument.makeGenesis(ceramic, { foo: 'bar' }, null)
+    const genesisCommit = await TileDocument.makeGenesis(ceramic.signer, { foo: 'bar' }, null)
```

Comment:
this still indicates there was a breaking change to our public API.  I think you updated the *most* of the methods of TileDocument, but not his one.  No test code calling methods on TileDocument should need to change as a result of this PR, if anywhere in this PR there is a change to code calling into TileDocument, that's a flag that we broke something.

Reply:
It actually works both ways now. Some tests use `ceramic` and some use `ceramic.signer` to verify that behavior.